### PR TITLE
Added permalink ability

### PR DESCRIPTION
**Pull request by jonnyhaynes**
As is standard with Twitter widgets, I've wrapped the Twitter posted time with a permalink. Can be controlled by by config `showPermalinks`, by default it's set to true.

### CODE REVIEW

File: js/twitterFetcher.js [hunk 1]
```diff
@@ -251,8 +260,13 @@
           }
           op += '<p class="tweet">' + strip(tweets[n].innerHTML) + '</p>';
           if (printTime) {
-            op += '<p class="timePosted">' +
-                times[n].getAttribute('aria-label') + '</p>';
+            if (permalinks) {
+                op += '<p class="timePosted"><a href="' + permalinksURL[n] + '">' +
```
Comment (jasonmayes) [inline js/twitterFetcher.js]: Please fix indentation here 2 space indent only for the if statement content...

**Comment by jasonmayes** [comment]
Thank you! I have added one minor comment to code which should be easy to address.

The only other thing: Could you update documentation by adding a @param comment in my exampleUsage.js file: https://github.com/jasonmayes/Twitter-Post-Fetcher/blob/master/js/exampleUsage.js#L37 explaining the variable name, its default value (true) and possible values. Eg something like:

 * @param object.showPermalinks [boolean] Set false if you don't want time to be permalinked.

**Comment by jonnyhaynes** [comment]
@jasonmayes I fixed the indentation and updated the docs as requested. Thanks.

**Comment by jasonmayes** [comment]
Thanks for contributing :-)

**Pull request merged**